tests/unit/amo/sagas/testLanding.js

```diff
@@ -156,7 +242,7 @@ describe(__filename, () => {
       const baseFilters = {
         addonType,
         category,
-        page_size: LANDING_PAGE_ADDON_COUNT,
+        page_size: LANDING_PAGE_EXTENSION_COUNT,
```

Comment:
Should this be `LANDING_PAGE_THEME_COUNT`?

Reply:
yes..:/